frame/contracts/src/wasm/runtime.rs

```diff
@@ -327,14 +327,14 @@ impl RuntimeCosts {
 			EcdsaRecovery => s.ecdsa_recover,
 			ChainExtension(amount) => amount,
 			#[cfg(feature = "unstable-interface")]
-			CallRuntime(weight) => weight,
+			CallRuntime(weight) => weight.ref_time(),
 			SetCodeHash => s.set_code_hash,
 			EcdsaToEthAddress => s.ecdsa_to_eth_address,
 		};
 		RuntimeToken {
 			#[cfg(test)]
 			_created_from: *self,
-			weight,
+			weight: Weight::from_ref_time(weight),
```

Comment:
It looks like to me that the weight could've been left as `weight`, and no changes here would be necessary.

Reply:
If changed, I get:

```
error[E0308]: mismatched types
   --> frame/contracts/src/wasm/runtime.rs:337:4
    |
337 |             weight,
    |             ^^^^^^ expected struct `frame_support::dispatch::Weight`, found `u64`
    ```

Reply:
Ah ok, I see what's going on here, because the other match arms above are returning a `u64` as well.